# DenseEnvironment.msg 

std_msgs/Header header
string input_mesh
int32 target_num_clusters
string output_mesh
string segmented_mesh_clusters
